monitor/documentation/pom.xml

```diff
@@ -31,40 +31,30 @@
 		</plugins>
 	</build>
 
-	<dependencies>
+    <dependencies>
         <!-- Spring -->
-		<dependency>
-			<groupId>org.springframework</groupId>
-			<artifactId>spring-webmvc</artifactId>
-		</dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+        </dependency>
 
         <!-- Servlet+JSP dependencies -->
-		<dependency>
-			<groupId>javax.servlet</groupId>
-			<artifactId>javax.servlet-api</artifactId>
-			<scope>provided</scope>
-		</dependency>
+        <dependency>
```

Comment:
In XML documents we have tabs instead of spaces (yes - it's an inconsistent rule, but let's keep the diffs consistent until the point where we decide to update the rule once and for all :))